# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/performance/PerformanceWorker.cpp
# Commit: 26d3176a00ef
# Full Hash: 26d3176a00ef338cef46f4ad6f8bdfb91f2f586c
# Author: Jens Stutte <jstutte@mozilla.com>
# Date: 2021-12-08 03:42:13
# Regressor Bug: 1744025
# File Overlap Count: 3
# Description:
#   Bug 1744025: Use CheckedUnsafePtr<WorkerPrivate> in WorkerGlobalScopeBase. r=dom-worker-reviewers,asuth
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D132709
# ==============================================================================

diff -r 2c8941fdcee5 -r 26d3176a00ef dom/performance/PerformanceWorker.cpp
--- a/dom/performance/PerformanceWorker.cpp	Tue Dec 07 16:47:14 2021 +0000
+++ b/dom/performance/PerformanceWorker.cpp	Tue Dec 07 16:47:15 2021 +0000
@@ -5,7 +5,7 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 #include "PerformanceWorker.h"
-#include "mozilla/dom/WorkerPrivate.h"
+#include "mozilla/dom/WorkerScope.h"
 #include "mozilla/StaticPrefs_dom.h"
 
 namespace mozilla::dom {